static/app/utils/requestError/sanitizePath.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Remove organization slugs from the path - we do not want them displayed in the Issues Stream (having them in issue details is ok)
+ */
+
+const SHORTENED_TYPE = {
+  organizations: 'org',
+  projects: 'project',
+  teams: 'team',
+};
+
+export function sanitizePath(path: string) {
+  return path.replace(
+    /(?<start>.*?)\/(?<type>organizations|projects|teams)\/(?<primarySlug>[^/]+)\/(?<contentType>[^/]+\/)?(?<tertiarySlug>[^/]+\/)?(?<end>.*)/,
+    function (...args) {
```

Comment:
Any reason you're writing this as a `function ()` over a fat arrow function?

Reply:
Nope